resources/assets/js/forms.js

```diff
@@ -254,6 +254,13 @@ ready(() => {
   document.querySelectorAll('[data-bs-toggle="popover"]').forEach((element) => new bootstrap.Popover(element));
 });
 
+/**
+ * Init Bootstrap Tooltips
+ */
+ready(() => {
+  document.querySelectorAll('[data-bs-toggle="tooltip"]').forEach((element) => new bootstrap.Tooltip(element));
```

Comment:
i would merge that with the popover init above

Reply:
@weeman1337 wants to do that later for the whole file